Carbon.Security/Carbon.Security.psm1

```diff
@@ -26,13 +26,13 @@ Set-StrictMode -Version 'Latest'
 $script:moduleRoot = $PSScriptRoot
 
 $psModulesDirPath = Join-Path -Path $script:moduleRoot -ChildPath 'Modules' -Resolve
-Import-Module -Name (Join-Path -Path $psModulesDirPath -ChildPath 'Carbon.Core') `
+Import-Module -Name (Join-Path -Path $psModulesDirPath -ChildPath 'Carbon.Core\Carbon.Core.psm1') `

```

Comment:
Do you need to add `"FlattenModules": true` to the `prism.json` for this to work?

Reply:
No. `FlattenModules` is true by default.